shell/platform/darwin/ios/BUILD.gn

```diff
@@ -25,6 +25,7 @@ shared_library("flutter_framework_dylib") {
     "framework/Headers/FlutterMacros.h",
     "framework/Headers/FlutterPlugin.h",
     "framework/Headers/FlutterViewController.h",
+    "framework/Headers/FlutterNavigationController.h",
```

Comment:
I believe you'll need to add the new files to some licenses index file as well. Travis would know for sure.